customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/service/SendPasswordWithSMS.java

```diff
@@ -44,21 +44,4 @@ public void sendPassword(Termination termination) {
         updateTermination.passwordSent();
```

Comment:
I know this is not part of the PR, but same as with the SendPackageNotification class, any changes to the domain should not be done within the infrastructure layer.